src/caughtup/__tests__/caughtUpReducer-test.js

```diff
@@ -1,6 +1,7 @@
 /* @flow strict-local */
 import deepFreeze from 'deep-freeze';
 
+import * as eg from '../../__tests__/lib/exampleData';
 import caughtUpReducer from '../caughtUpReducer';
 import { MESSAGE_FETCH_START, MESSAGE_FETCH_COMPLETE } from '../../actionConstants';
```

Comment:
Looks like the MESSAGE_FETCH_COMPLETE import isn't used anymore.

EDIT: Ah, looks like it was removed in the next commit. Let's remove it in the same commit that removes all uses, so `tools/test lint` passes at each commit. I caught this with `git rebase -i --exec 'tools/test --diff HEAD^`, which checks every commit on the branch after `upstream/master` (if the branch is set to track `upstream/master`). GitHub only runs CI tests for the single commit at the tip of the branch, I guess to save resources.

Reply:
Thanks, I'll keep this in mind. You had mentioned this before, but I forgot to use it.